### PR TITLE
Adding --path.rootfs to node exporter arguments and upgrading nodeexporter

### PR DESCRIPTION
**Pull request by imrajdas**
Motivation:
- To show the prometheus metrics of /mnt/disks/ssd0 which is mounted on filesystem `/dev/sdb`

Changes:
- Adding `--path.rootfs=/host/root` to node exporter args
- Adding volumeMount `root` which is mount on /host/root
- Upgrading node exporter to 1.0.1 to support `--path.rootfs`

 Issue: https://github.com/prometheus/test-infra/issues/159 https://github.com/prometheus/test-infra/issues/328
Signed-off-by: Raj Babu Das <mail.rajdas@gmail.com>

### CODE REVIEW

**Comment by imrajdas** [comment]
cc @geekodour

File: prombench/manifests/prombench/benchmark/4_node-exporter.yaml [hunk 1]
```diff
@@ -110,10 +110,11 @@ spec:
       nodeSelector:
         node-name: prometheus-{{ .PR_NUMBER }}
       containers:
-      - image: quay.io/prometheus/node-exporter:v0.16.0
+      - image: quay.io/prometheus/node-exporter:v0.18.0
         args:
         - "--path.procfs=/host/proc"
         - "--path.sysfs=/host/sys"
+        - "--path.rootfs=/host/root"
```
Comment (geekodour) [inline prombench/manifests/prombench/benchmark/4_node-exporter.yaml:126]: There are quite a few unnecessary things here. we could use `--collector.filesystem.ignored-mount-points` and `--collector.filesystem.ignored-fs-types` flags to get rid of those(whichever suites). (from the Disk Space Used Panel)

di

**Review by geekodour — COMMENTED** [review]
Nice work! :)

Just some suggestions and changes needed to get the dashboards to show the required info.

File: prombench/manifests/prombench/benchmark/4_node-exporter.yaml [hunk 2]
```diff
@@ -126,6 +127,10 @@ spec:
         - name: sys
           readOnly: true
           mountPath: /host/sys
+        - name: root
+          mountPropagation: HostToContainer
```
[the remark this answers is not in the document]
Comment (imrajdas) [reply prombench/manifests/prombench/benchmark/4_node-exporter.yaml]: We don't need it, I tested it without `mountPropagation`. It works fine

File: prombench/manifests/prombench/benchmark/4_node-exporter.yaml (hunk 1 above)
Comment (imrajdas) [reply prombench/manifests/prombench/benchmark/4_node-exporter.yaml:126]: Added the ignore flags for some disks

**Comment by geekodour** [comment]
Can we add the changes of https://github.com/prometheus/test-infra/pull/424 here aswell? and close  #424

**Comment by imrajdas** [comment]
Disk Space Used Basic Dashboard
Selection_008

**Comment by imrajdas** [comment]
File system fullness dashboard
prom-exporter-2

**Pull request merged**